index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+      integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <nav>
+      <a href="#">My Logo</a>
+      <div id="list">
+        <ul id="list-2">
+          <li id="portfolio">Portfolio</li>
+          <li id="about">About</li>
+          <li id="contact">Contact</li>
+        </ul>
+      </div>
+    </nav>
+    <header>
+      <h1>
+        I'm Abdul Hadi Jafari <br />
+        Glad to see you!
+      </h1>
+      <p>
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social">
+        <h2>LET'S CONNECT</h2>
+        <ul class="social-icons">
+          <li><i class="fa-brands fa-twitter"></i></li>
+          <li><i class="fa-brands fa-linkedin"></i></li>
+          <li><i class="fa-brands fa-medium"></i></li>
+          <li><i class="fa-brands fa-github"></i></li>
+          <li><i class="fa-brands fa-angellist"></i></li>
+        </ul>
```

Comment:
- [ ] Please notice in this `.social-icons` section that the color of the icons is black however in the reference you can notice that they are using a color of `#505F79`, please make this change.
 
Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/155594545-bdfbecf5-0d8c-41bd-9505-1478e7d9cd64.png)|![image](https://user-images.githubusercontent.com/18730106/155594610-a76f4b70-31be-4d3e-a015-7e206d56a4dd.png)